llvm/test/LTO/AMDGPU/gpu-rdc-amdgpu-attrs.ll

```diff
@@ -0,0 +1,12 @@
+; RUN: opt -O3 -debug-only=amdgpu-attributor -S -o - %s 2>&1 | FileCheck %s --check-prefix=PRE-LINK
+; RUN: opt -passes="lto<O3>" -debug-only=amdgpu-attributor -S -o - %s 2>&1 | FileCheck %s --check-prefix=POST-LINK
```

Comment:
`-debug-only` won't work wth `LLVM_ENABLE_ASSERTIONS=OFF`.

Reply:
I see, do you know how I could add a requirement for assertions then?

Reply:
Created a PR: #106027